README.md

```diff
@@ -9,3 +9,9 @@ GitHub pages for IRIS-HEP website
 Do not make a PR to the output repository directly! Instead, make pull requests to the [iris-hep.github.io-source repository](https://github.com/iris-hep/iris-hep.github.io-source/). The advantage of making a pull request instead of directly editing is that changes made via pull request will be tested to ensure that all is ok with your changes...
 
 
+## testing locally:
+
+```
+gem install bundler jekyll
```

Comment:
```suggestion
gem install bundler
```

You shouldn't install Jekyll here. Just bundler, then bundler will handle the rest.